### PR TITLE
Fixed emoji removal

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: src/AppBundle/Admin/PageAdmin.php [hunk 1]
```diff
@@ -22,16 +22,19 @@ protected function configureFormFields(FormMapper $formMapper)
         if ($this->getSubject()->getId() === null) {
             $formMapper->add('title', null, [
```
Comment (tgalopin) [inline src/AppBundle/Admin/PageAdmin.php]: TextType?

File: src/AppBundle/Entity/SocialShareCategory.php [hunk 2]
```diff
@@ -45,28 +44,28 @@ class SocialShareCategory
...
+        return $this->name;
     }
 
     public function getId(): int
     {
         return $this->id;
     }
 
-    public function getName(): ?string
+    public function getName(): string
```
Comment (tgalopin) [inline src/AppBundle/Entity/SocialShareCategory.php]: I think this won't always work (when the form will put null data in the name property).

File: src/AppBundle/Admin/PageAdmin.php (hunk 1 above)
Comment (HeahDude) [inline src/AppBundle/Admin/PageAdmin.php]: Good catch!

File: src/AppBundle/Entity/SocialShareCategory.php (hunk 2 above)
Comment (HeahDude) [inline src/AppBundle/Entity/SocialShareCategory.php]: Yes that's why there is the cast below ;).

**Pull request merged**